homeassistant/components/automation/template.py

```diff
@@ -28,7 +29,9 @@
 )
 
 
-async def async_attach_trigger(hass, config, action, automation_info):
+async def async_attach_trigger(
+    hass, config, action, automation_info, *, platform_type: str = "state"
```

Comment:
This is wrong. You default to "state" but it should be "template"

Reply:
Reverted (no longer using template trigger)